src/components/ReleaseMonitor/SearchFilter.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { SearchInput } from '@patternfly/react-core';
+
+interface SearchFilterProps {
+  value: string;
+  onChange: (value: string) => void;
+}
+
+export const SearchFilter: React.FC<SearchFilterProps> = ({ value, onChange }) => {
+  return (
+    <SearchInput
+      placeholder="Search by release plan"
+      value={value}
+      onChange={(_, v) => onChange(v)}
```

Comment:
you need to debounce this call for performance purposes otherwise every typed word will cause state change

Reply:
I updated the PR, please review.